policyengine_us/variables/gov/states/mt/tax/income/credits/mt_elderly_homeowner_or_renter_credit/mt_gross_household_income.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class mt_gross_household_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana gross household income for the elderly homeowner/renter credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    adds = "gov.states.mt.tax.income.credits.elderly_homeowner_or_renter_credit.gross_income_sources"
+    # Tax form specifies total social social security - the taxable social security amounts
```

Comment:
delete this through 15. it's correctly added to gross_income_sources as to capture total SS benefits: taxable via AGI, plus tax-exempt